lib/jobs/node-discovered-alert-job.js

```diff
@@ -71,5 +80,19 @@ function nodeAlertJobFactory(
         });
     };
 
+    NodeAlertJob.prototype._getHwInfoCatalog = function () {
+        var self = this;
+        return waterline.catalogs.findMostRecent({
+            node: self.nodeId,
+            source: 'dmi'
+        }).then(function (catalog) {
+            if (!catalog) {
+                return Promise.reject(
+                    new Error('Could not find dmi catalog data.'));
```

Comment:
@keedya mentioned in slack we probably don't want to fail here if the data isn't available. I can clean this up tomorrow to make sure missing catalog data doesn't fail this job.